Moblin/Various/Model.swift

```diff
@@ -4962,6 +4977,32 @@ extension Model {
             self.remoteControlGeneral = general
             self.remoteControlTopLeft = topLeft
             self.remoteControlTopRight = topRight
+            if self.isRemoteControlAssistantConnected() {
```

Comment:
This check can be removed as the same check is done in the beginning of the function.

Reply:
done